<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Proteomics</title>
  <meta name="description" content="Notes on Proteomics">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Proteomics" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Notes on Proteomics" />
  <meta name="github-repo" content="mstaniak/proteomics" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Proteomics" />
  
  <meta name="twitter:description" content="Notes on Proteomics" />
  

<meta name="author" content="Mateusz Staniak">


<meta name="date" content="2019-03-04">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="proteomics-software.html">
<link rel="next" href="analysis-of-mass-spectrometry-data.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Proteomics</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="about-this-book.html"><a href="about-this-book.html"><i class="fa fa-check"></i><b>1</b> About this book</a></li>
<li class="chapter" data-level="2" data-path="proteomics.html"><a href="proteomics.html"><i class="fa fa-check"></i><b>2</b> Proteomics</a><ul>
<li class="chapter" data-level="2.1" data-path="proteomics.html"><a href="proteomics.html#what-is-proteomics"><i class="fa fa-check"></i><b>2.1</b> What is Proteomics?</a></li>
<li class="chapter" data-level="2.2" data-path="proteomics.html"><a href="proteomics.html#why-do-we-need-proteomics"><i class="fa fa-check"></i><b>2.2</b> Why do we need Proteomics?</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="proteins.html"><a href="proteins.html"><i class="fa fa-check"></i><b>3</b> Proteins</a></li>
<li class="chapter" data-level="4" data-path="proteomics-databases.html"><a href="proteomics-databases.html"><i class="fa fa-check"></i><b>4</b> Proteomics Databases</a><ul>
<li class="chapter" data-level="4.1" data-path="proteomics-databases.html"><a href="proteomics-databases.html#raw-mass-spectrometry-data"><i class="fa fa-check"></i><b>4.1</b> Raw Mass Spectrometry data</a><ul>
<li class="chapter" data-level="4.1.1" data-path="proteomics-databases.html"><a href="proteomics-databases.html#massive"><i class="fa fa-check"></i><b>4.1.1</b> MassIVE</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="proteomics-databases.html"><a href="proteomics-databases.html#protein-quantification-and-identification"><i class="fa fa-check"></i><b>4.2</b> Protein Quantification and Identification</a></li>
<li class="chapter" data-level="4.3" data-path="proteomics-databases.html"><a href="proteomics-databases.html#protein-knowledge"><i class="fa fa-check"></i><b>4.3</b> Protein Knowledge</a><ul>
<li class="chapter" data-level="4.3.1" data-path="proteomics-databases.html"><a href="proteomics-databases.html#uniprot"><i class="fa fa-check"></i><b>4.3.1</b> UniProt</a></li>
<li class="chapter" data-level="4.3.2" data-path="proteomics-databases.html"><a href="proteomics-databases.html#nextprot"><i class="fa fa-check"></i><b>4.3.2</b> NextProt</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="proteomics-software.html"><a href="proteomics-software.html"><i class="fa fa-check"></i><b>5</b> Proteomics software</a></li>
<li class="chapter" data-level="6" data-path="mass-spectrometry-based-proteomics.html"><a href="mass-spectrometry-based-proteomics.html"><i class="fa fa-check"></i><b>6</b> Mass Spectrometry - Based Proteomics</a><ul>
<li class="chapter" data-level="6.1" data-path="mass-spectrometry-based-proteomics.html"><a href="mass-spectrometry-based-proteomics.html#ms-based-proteomics-introduction"><i class="fa fa-check"></i><b>6.1</b> MS-Based Proteomics Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="mass-spectrometry-based-proteomics.html"><a href="mass-spectrometry-based-proteomics.html#mass-spectrometer"><i class="fa fa-check"></i><b>6.2</b> Mass Spectrometer</a></li>
<li class="chapter" data-level="6.3" data-path="mass-spectrometry-based-proteomics.html"><a href="mass-spectrometry-based-proteomics.html#sample-preparation"><i class="fa fa-check"></i><b>6.3</b> Sample Preparation</a><ul>
<li class="chapter" data-level="6.3.1" data-path="mass-spectrometry-based-proteomics.html"><a href="mass-spectrometry-based-proteomics.html#complete-digestion"><i class="fa fa-check"></i><b>6.3.1</b> Complete Digestion</a></li>
<li class="chapter" data-level="6.3.2" data-path="mass-spectrometry-based-proteomics.html"><a href="mass-spectrometry-based-proteomics.html#protein-and-peptide-separation"><i class="fa fa-check"></i><b>6.3.2</b> Protein and Peptide Separation</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="mass-spectrometry-based-proteomics.html"><a href="mass-spectrometry-based-proteomics.html#ionization-methods-and-mass-analyzers"><i class="fa fa-check"></i><b>6.4</b> Ionization Methods and Mass Analyzers</a><ul>
<li class="chapter" data-level="6.4.1" data-path="mass-spectrometry-based-proteomics.html"><a href="mass-spectrometry-based-proteomics.html#maldi-tof-ms"><i class="fa fa-check"></i><b>6.4.1</b> MALDI-TOF MS</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="mass-spectrometry-based-proteomics.html"><a href="mass-spectrometry-based-proteomics.html#fragmentation-methods"><i class="fa fa-check"></i><b>6.5</b> Fragmentation Methods</a></li>
<li class="chapter" data-level="6.6" data-path="mass-spectrometry-based-proteomics.html"><a href="mass-spectrometry-based-proteomics.html#detectors"><i class="fa fa-check"></i><b>6.6</b> Detectors</a></li>
<li class="chapter" data-level="6.7" data-path="mass-spectrometry-based-proteomics.html"><a href="mass-spectrometry-based-proteomics.html#ms-ms-spectra"><i class="fa fa-check"></i><b>6.7</b> MS-MS Spectra</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="analysis-of-mass-spectrometry-data.html"><a href="analysis-of-mass-spectrometry-data.html"><i class="fa fa-check"></i><b>7</b> Analysis of Mass Spectrometry data</a><ul>
<li class="chapter" data-level="7.1" data-path="analysis-of-mass-spectrometry-data.html"><a href="analysis-of-mass-spectrometry-data.html#data-formats"><i class="fa fa-check"></i><b>7.1</b> Data Formats</a><ul>
<li class="chapter" data-level="7.1.1" data-path="analysis-of-mass-spectrometry-data.html"><a href="analysis-of-mass-spectrometry-data.html#raw-data-formats"><i class="fa fa-check"></i><b>7.1.1</b> Raw Data Formats</a></li>
<li class="chapter" data-level="7.1.2" data-path="analysis-of-mass-spectrometry-data.html"><a href="analysis-of-mass-spectrometry-data.html#processed-data-formats"><i class="fa fa-check"></i><b>7.1.2</b> Processed Data Formats</a></li>
<li class="chapter" data-level="7.1.3" data-path="analysis-of-mass-spectrometry-data.html"><a href="analysis-of-mass-spectrometry-data.html#data-import"><i class="fa fa-check"></i><b>7.1.3</b> Data Import</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="analysis-of-mass-spectrometry-data.html"><a href="analysis-of-mass-spectrometry-data.html#data-analysis"><i class="fa fa-check"></i><b>7.2</b> Data Analysis</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="protein-sequence-analysis.html"><a href="protein-sequence-analysis.html"><i class="fa fa-check"></i><b>8</b> Protein sequence analysis</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Proteomics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="mass-spectrometry---based-proteomics" class="section level1">
<h1><span class="header-section-number">Chapter 6</span> Mass Spectrometry - Based Proteomics</h1>
<p>This chapter is based on TODO: cytowanie do “Introduction to Mass Spectrometry-Based Proteomics” Matthiesen, Bunkenborg. TODO: uzupełnić w oparciu o wykłady Dirka.</p>
<div id="ms-based-proteomics-introduction" class="section level2">
<h2><span class="header-section-number">6.1</span> MS-Based Proteomics Introduction</h2>
<p>Mass spectrometry is one of the most important methods of studying proteins. It provides information that is complementary to information that can be obtained using techniques such as NMR or X-ray crystallography. Mass spectrometry is easier to automatate (and is more reproducible), more sensitive, can be used on complex mixtures and is easy to combine with different separation techniques. One of the main goals of MS-based Proteomics is identification and quantification of proteins and protein mixtures. Post translational modifications resulting from many processes such as phosphorylation, acetylation or methylation can be analyzed by MS. MS can be also used to determine N- or C-terminus of proteins to examine the primary structure of proteins or map proteolytic cleavage sites in proteins. Moreover, MS can be applied to determine if the protein folded correctly and to other structural studies.</p>
<p>These notes focus on protein identification and quantification. There are three main approach to MS-based protein identification:</p>
<ul>
<li><strong>bottom-up approach</strong>, in which proteins are cut into peptides that serve as input to Mass Spectrometer,</li>
<li><strong>shotgun approach</strong>, which is a special case of bottom-up proteomics where a complex mixture of proteins is digested into peptides (usually trypsin is used for this task) which are then separated using Liquid Chromatography,</li>
<li><strong>top-down approach</strong>, in which the mass spectrometer isolates a full length protein (which can be fragmented inside the mass spectrometer and the masses of fragments can be also recorded).</li>
</ul>
<p>TODO: work flow identyfikacji TODO: dobra ilustracja tych trzech strategii TODO: co to jest LC? Co daje, co zmienia?</p>
</div>
<div id="mass-spectrometer" class="section level2">
<h2><span class="header-section-number">6.2</span> Mass Spectrometer</h2>
<p>A mass spectrometer is a device for measuring the mass-to-charge ratio of ionized molecules. All mass spectrometers consist of three main parts: - an ion source, where analyte ions are produced by transferring molecules from the condensed (liquid or solid) state to gas phase (ionization is done in the process and molecule can receive either positive or negative charge). Different methods can be used for ionization, most common in proteomics are ESI (Electro Spray Ionization) and MALDI (Matrix-Assisted Laser Desorption and Ionization), TODO: lepsze slowo niz “receive” - a mass analyzer, where the ions are separated according to their mass to charge ration (<em>m/z</em>). Multiple different types of analyzer exists and they can be used in combination (by tandem mass spectrometers) - a detector, which records a signal that can be electronically amplified and stored. TODO: opis detektora</p>
<p>TODO: dlaczego w ogole m/z jest pomocne.</p>
<p>The results from mass analyzers can be transformed to internsities as a function of <em>m/z</em> (for example by Fourier Transform). The <em>m/z</em> vs intensity plot is a mass spectrum that is the final output from the mass spectrometer.</p>
<p>Many combinations of instrument setups and fragmentation techniques can be used for MS-based Proteomics. These choices affect the data analysis process, so understanding how the data were collected is crucial to protein identification and quantification.</p>
</div>
<div id="sample-preparation" class="section level2">
<h2><span class="header-section-number">6.3</span> Sample Preparation</h2>
<p>The quality of MS data depends on sample complexity and purity. Main issues are caused by - ion suppression, where many different species compete for the charges during the ionization process, TODO: wiecej, to jest tylko w LC czy ogolnie? - contamination associated with the mass spectrometer (for example oil used for mass spectrometer pumps), sample preparation in the lab (for example sheep keratin from clothing) or earlier sample preparation (for example serum used to supplement growth media when studying proteins from cell cultures). Even if contaminating proteins are included in the database query after MS experiment to avoid identifying proteins homologous to proteins from contamination, problems with ion suppresion or overlap with target peptides of interest may occur.</p>
<p>TODO: dodac ogolne uwagi z 2.2 Powiazac je z przykladami Dirka z trypsyna.</p>
<div id="complete-digestion" class="section level3">
<h3><span class="header-section-number">6.3.1</span> Complete Digestion</h3>
<p>Peptide and protein indentification methods are often robust against partial incomplete digestion. TODO: dodac przyklad z 2.3 z artykulu, o ile bedzie to zrozumiale. On the other hand, some relative and absolute quantitation techniques are affected by uncontrolled digestion. The sensitive methods include iCAT, HysTag, iTRAQ and label free quantitation (relative measure); AQUA and QCONCAT (absolute quantification). SILAC (relative) and PSAQ (absolute) are robust methods.</p>
</div>
<div id="protein-and-peptide-separation" class="section level3">
<h3><span class="header-section-number">6.3.2</span> Protein and Peptide Separation</h3>
<p>For simple peptide sample is associated with less ion suppresion, which causes smaller risk of peak overlap and more sensitivity. Protein and peptide separation techniques can be applied to obtain simple peptide mixtures.</p>
<p>TODO: metody separacji z 2.4, o ile to bedzie zrozumiale, ew. powiazac z wykladami Dirka</p>
<p>Before LC-MS, the peptide sample is usually cleaned up (desalted and cleaned from gel debris) by reversed phase columns.</p>
<p>TODO: co to jest column &lt;…&gt; ?</p>
</div>
</div>
<div id="ionization-methods-and-mass-analyzers" class="section level2">
<h2><span class="header-section-number">6.4</span> Ionization Methods and Mass Analyzers</h2>
<p>The formation of gas-phase ions is required before the molecular masses of a sample can be measured by the mass analyzer. ESI and MALDI are two methods that are able to eficiently create the ions from large biomolecules with minimum fragmentation. Currently, five mass analyzer types are commonly used: quadrupole mass filters (Q), time of flight mass analyzers (TOF), ion traps (IT), FT-ICR and Orbitrap analyzers. They can work as standalone but they are usually combined in hybrid configurations of which some are more popular. The most populars configurations include Q-TOF, Q-ion mobility-TOF, triple quadrupole, ion trap-Orbitrap and Q-Orbitrap.</p>
<div id="maldi-tof-ms" class="section level3">
<h3><span class="header-section-number">6.4.1</span> MALDI-TOF MS</h3>
<div id="maldi-ionization" class="section level4">
<h4><span class="header-section-number">6.4.1.1</span> MALDI Ionization</h4>
<p>In MALDI, the analyte is mixed with a compound (called the matrix, usually small aromatic acids), which absorbs the energy from the laser. The sample is co-crystallized with an excess amount of the matrix. Irradiation with a short-pulsed laser causes mainly ionization of the matrix followed by energy and proton transfer to the analyte. MALDI efficiently ionizes peptides and protein intact. This method is robust t typical contaminants present in protein/peptide samples such as salts, urea or glycerol, though it is sensitive to low concentrations of SDS.</p>
<p>Matrix preparation affects the quality of MALDI-MS spectra. There is no universal matrix preparation procedure. While several general guidelines have been found, several combinations should be tested for each sample and some optimization specific to the sample may be needed.</p>
<p>TODO: jasniejszy/bardziej szczegolowy opis</p>
<p>TODO: co to dokladnie PMF?</p>
</div>
<div id="the-tof-mass-analyzer" class="section level4">
<h4><span class="header-section-number">6.4.1.2</span> The TOF Mass Analyzer</h4>
<p>In TOF-MS a population of ions, for example derived by MALDI, is accelerated by an eletrical potential and each ion acquires a kinetic energy depending on its mass and charge state. After acceleration the ions pass through a field free region where each ion is traveling with a speed characteristic of their <em>m/z</em> value. At the end of the field free regiona detector measures the time of flight (TOF). TOF is a sum of flight times in two regions - accelaration region and field free region - and detection time. TOF in field free region can be estimated by the drift time, which is proportional to the square root of <em>m/z</em>.</p>
<p>TODO: wzor na drift time</p>
<p>TODO: uwagi nt mass resolution (tu lub gdzies indziej)</p>
</div>
<div id="maldi-tof-ms-spectrum" class="section level4">
<h4><span class="header-section-number">6.4.1.3</span> MALDI-TOF MS Spectrum</h4>
<p>TOF recored by MALDI-TOF spectrometer must be transformed to relative intensity.</p>
</div>
<div id="maldi-tof-ms-issues" class="section level4">
<h4><span class="header-section-number">6.4.1.4</span> MALDI-TOF MS Issues</h4>
<p>Issues with MALDI include the fact that some peptides may not be able to co-crystallize with the used matrix and arginine-containing peptides are more frequently observed in MALDI MS spectra due to higher proton affinity and higher stability. Due to the competition for the protons in the matrix plume, a limited protein sequence coverage is observed (30-40% in good samples). Moreover, heterogenity in the matrix-sample sample crystal formation can cause difficulties for quantifying and reproducing results. The major problem with MALDI-TOF MS is obtaining highly significant search results due to not all expected tryptic peptides showing up in experimental MS spectrum and presence of unknown mass peaks. PMF is limited to the analysis of purified proteins.</p>
</div>
</div>
</div>
<div id="fragmentation-methods" class="section level2">
<h2><span class="header-section-number">6.5</span> Fragmentation Methods</h2>
</div>
<div id="detectors" class="section level2">
<h2><span class="header-section-number">6.6</span> Detectors</h2>
</div>
<div id="ms-ms-spectra" class="section level2">
<h2><span class="header-section-number">6.7</span> MS-MS Spectra</h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="proteomics-software.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="analysis-of-mass-spectrometry-data.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
